### PR TITLE
Create AWSCloud.com.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/AWSCloud.com.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+	For other Amazon coverage, see Amazon.xml.
```
Comment (J0WI) [inline src/chrome/content/rules/AWSCloud.com.xml]: See also `email.awscloud.com`

Comment (koops76) [inline src/chrome/content/rules/AWSCloud.com.xml]: Dropped it seems.

Comment (J0WI) [inline src/chrome/content/rules/AWSCloud.com.xml]: Could you add a comment for it?

Comment (koops76) [inline src/chrome/content/rules/AWSCloud.com.xml]: @J0WI Done.

**Pull request merged**